bin/sdk/gutenberg.js

```diff
@@ -4,20 +4,51 @@
  */
 const fs = require( 'fs' );
 const path = require( 'path' );
+const reduce = require( 'lodash/reduce' );
+const pickBy = require( 'lodash/pickBy' );
+const isEmpty = require( 'lodash/isEmpty' );
 
 exports.config = ( { argv: { inputDir, outputDir }, getBaseConfig } ) => {
 	const baseConfig = getBaseConfig( {
 		cssFilename: '[name].css',
 		externalizeWordPressPackages: true,
 	} );
-	const editorScript = path.join( inputDir, 'editor.js' );
-	const viewScript = path.join( inputDir, 'view.js' );
+
+	const presetPath = path.join( inputDir, 'index.json' );
+	const utilsPath = path.join( inputDir, 'utils' );
+
+	let viewBlocksPoints;
+	let viewScriptEntry;
+	if ( fs.existsSync( presetPath ) ) {
+
+		const preset = require( presetPath );
+
+	  // Helps split up each block into its own folder view script 
+		viewBlocksPoints = pickBy( reduce( preset, (obj, block) => ( { [ block + '/view'] : [path.join( inputDir, '../../'+ block +'/view.js' )]  } )  ), fs.existsSync );
+
+		utilScripts = fs.readdirSync( utilsPath ).map( utilFile => path.join( utilsPath, utilFile ) ).filter( fs.existsSync );
```

Comment:
I wonder if we could simply import utils in each block depending on them.

Right now building a single block is a bit tricky because many of them depend on `jetpack` category which gets set only by importing the jetpack-preset...

Reply:
I am just trying to make sure that we have all the same code as before.

Reply:
Sure! Makes sense for this PR. :-)